src/Razor/src/Microsoft.VisualStudio.RazorExtension/language-configuration.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "comments": {
+    "blockComment": [ "@*", "*@" ]
+  },
+  "brackets": [
+    ["<!--", "-->"],
+    ["<", ">"],
+    ["{", "}"],
+    ["(", ")"]
+  ],
+  "autoClosingPairs": [
+    { "open": "{", "close": "}"},
+    { "open": "[", "close": "]"},
+    { "open": "(", "close": ")" },
+    { "open": "'", "close": "'" },
+    { "open": "\"", "close": "\"" },
```

Comment:
Why not use the `"notIn": ["string", "comment"] }` property?

Reply:
The bulk of this file was originally from the VSCode impl of Razor support and they didn't have it? We could and probably should utilize that notIn description

Reply:
Will address this if we get reports from users purely from the perspective of keeping the file as consistent as possible with what we've already proven to work in VSCode